packages/block-editor/src/components/link-control/index.js

```diff
@@ -187,7 +187,15 @@ function LinkControl( {
 	const wrapperNode = useRef();
 	const textInputRef = useRef();
 	const searchInputRef = useRef();
-	const isEndingEditWithFocusRef = useRef( false );
+	// TODO: Remove entityUrlFallbackRef and previewValue in favor of value prop after taxonomy entity binding
```

Comment:
Thanks for documenting this here, it will make it a lot easier to remove it when the time comes.